kopf/reactor/invocation.py

```diff
@@ -41,39 +41,25 @@ def context(
         for var, token in reversed(tokens):
             var.reset(token)
 
-
-async def invoke(
-        fn: Invokable,
-        *args: Any,
-        cause: Optional[causation.BaseCause] = None,
-        **kwargs: Any,
-) -> Any:
+def get_invoke_arguments(*args: Any, cause: Optional[causation.BaseCause] = None, **kwargs: Any) -> Dict[str, Any]:
```

Comment:
Can you please put each arg here (and the resulting type) to its own line — to be consistent with the rest of the codebase?

Reply:
Fixed in last push